CODE_OF_CONDUCT.md

```diff
@@ -26,7 +26,7 @@ Examples of unacceptable behavior by participants include:
   advances
 * Trolling, insulting/derogatory comments, and personal or political attacks
 * Public or private harassment
-* Publishing others' private information, such as a physical or electronic
+* Publishing other's private information, such as a physical or electronic
```

Comment:
'others' is the plural form, so the possessive apostrophe comes after the "s" and not before